azure/notebooks/remote-explanation/azure-gpu-shap.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>I'd also link to the README for these setup steps on Azure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>